src/plugins/vis_type_table/public/components/table_vis_component_group.tsx

```diff
@@ -23,7 +23,7 @@ export const TableVisComponentGroup = memo(
     return (
       <>
         {tableGroups.map(({ table, title }) => (
-          <EuiFlexItem key={title} className="visTable__group">
+          <EuiFlexItem key={title} className="visTable__group" grow={false}>
```

Comment:
Do we want to use `flex-grow` instead of `grow` here?

Reply:
I believe they are the same, it just that the `grow` is prop of `OuiFlexItem` that sets the `flex-grow` css property of div element within to specific value.